python/tidy/servo_tidy/tidy.py

```diff
@@ -450,6 +450,8 @@ def check_rust(file_name, lines):
     prev_crate = {}
     prev_mod = {}
     prev_feature_name = ""
+    indent = 0
+    prev_indent = 0
```

Comment:
Having `prev_indent`, `indent`, and `current_indent` is kind of irking me. I'm definitely open to suggestions.